com.unity.render-pipelines.high-definition/Editor/Material/TerrainLit/TerrainLitGUI.cs

```diff
@@ -311,8 +311,8 @@ bool ITerrainLayerCustomUI.OnTerrainLayerGUI(TerrainLayer terrainLayer, Terrain
                             float amplitude = Mathf.Max(maskMapRemapMax.z - maskMapRemapMin.z, Mathf.Epsilon); // to avoid divide by zero
                             float heightBase = -maskMapRemapMin.z / amplitude;
                             amplitude = EditorGUILayout.FloatField(styles.heightAmplitude, amplitude * 100) / 100;
-                            heightBase = EditorGUILayout.FloatField(styles.heightBase, heightBase * 100) / 100;
-                            maskMapRemapMin.z = heightBase * amplitude;
+                            heightBase = EditorGUILayout.Slider(styles.heightBase, heightBase, 0.0f, 1.0f);
+                            maskMapRemapMin.z = -heightBase * amplitude;
```

Comment:
The heart of the PR. Missing sign caused the Amplitude -> Min/Max parametrization to go A-wire. Works as designed now.